specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/ResourceProvider.json

```diff
@@ -0,0 +1,1398 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-09-01",
+    "title": " API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Web/generateGithubAccessTokenForAppserviceCLI": {
```

Comment:
Do you also have ARM manifest changes to support this action? Can you please provide a PR link?

Reply:
ARM Manifest change: 
https://msazure.visualstudio.com/One/_git/AAPT-Antares-Websites/pullrequest/3700170?_a=files&path=%2Fsrc%2FHosting%2FAdministrationService%2FMicrosoft.Web.Hosting.Administration.GeoScale.Csm%2FCSMManifests%2FMicrosoft.Web.Manifest-prod.json

Reply:
Thank you. generateGithubAccessTokenForAppserviceCLI is modeled as a tracked resource in the manifest. Tracked resources must implement GET, PUT, PATCH, DELETE & LIST. 
https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc006-tracked-resource-type-supports-get-put-patch-delete--list-at-all-scopes-as-applicable

I think it should be a proxy resource.

Reply:
good catch, thanks. Changed to proxy

Reply:
Updated manifest since we're bumping all API's now